deploy/handler/role.yaml

```diff
@@ -76,10 +76,9 @@ rules:
   - pods
   - nodes
   - namespaces
+  - secrets
   verbs:
-  - get
-  - list
-  - watch
+  - '*'
```

Comment:
We only need to create secrets, rights? Please add a new item in this list specifically for secrets and verbs they need.